ra_aid/models_params.py

```
@@ -27,6 +27,9 @@
         "gpt-4o-mini": {"token_limit": 128000, "supports_temperature": True},
         "o1-preview": {"token_limit": 128000, "supports_temperature": False},
         "o1-mini": {"token_limit": 128000, "supports_temperature": False},
+        "o1-preview": {"token_limit": 128000, "supports_temperature": False},
```

Ruff (0.8.2)

Dictionary key literal `"o1-preview"` repeated

Remove repeated key literal `"o1-preview"`

(F601)